db/seeds.rb

```diff
@@ -48,8 +48,7 @@
                    first_name: 'Mr.', last_name: 'Fake', role: 5)
 
 (0...titles.count).each do |i|
- article = Article.create(title: titles[i], teaser: teasers[i], body: body[i], category: categories[rand(categories.count - 1)],
-                 user_id: user.id)
+ article = Article.create(title: titles[i], teaser: teasers[i], body: body[i], category: categories[rand(categories.count - 1)], user_id: user.id, premium: [true, false].sample)
```

Comment:
what is this .sample doing?

Reply:
sample is the ruby method you can use on any array. It picks one random value from the array. In this case it randomly picks one of 'true' or 'false'. This way when we seed, premium attribute will be assigned randomly to different articles

Reply:
Allright 👍